.github/workflows/eqc.yaml

```diff
@@ -3,8 +3,8 @@ name: EQC Tests
 on: [push]
 
 jobs:
-  eqc-tests:
-    if: github.repository_owner == 'tremor-rs'
+  api:
+    if: github.repository_owner == 'tremor-rs' && github.actor!= 'dependabot[bot]'
```

Comment:
🧙

Reply:
:D lets hope this one works, and dependabot shows nice and green when we only have EQC failing (or skipping)